kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/internal/BaseOperation.java

```diff
@@ -216,6 +214,9 @@ public R withName(String name) {
 
   @Override
   public ExtensibleResource<T> lockResourceVersion(String resourceVersion) {
+    if (resourceVersion == null) {
```

Comment:
`Utils.isNotNullOrEmpty` instead?

Reply:
https://github.com/fabric8io/kubernetes-client/pull/4027#discussion_r848315325 - should apply here as well.